Description: <blankon-patch>
 Patch for gnome-screenserver in blankon.
 .
 gnome-screensaver (3.6.1-7blankon3) uluwatu; urgency=medium
 .
   * Fix patch
Author: AnTo SamalonA <ansamsoftdev@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2017-11-19

--- gnome-screensaver-3.6.1.orig/src/gnome-screensaver.desktop.in
+++ gnome-screensaver-3.6.1/src/gnome-screensaver.desktop.in
@@ -5,7 +5,7 @@ _Comment=Launch screensaver and locker p
 Icon=preferences-desktop-screensaver
 Exec=gnome-screensaver
 OnlyShowIn=GNOME;
-AutostartCondition=GNOME3 if-session gnome-fallback
+AutostartCondition=GNOME3 unless-session gnome
 NoDisplay=true
 X-GNOME-Autostart-Phase=Application
 X-GNOME-Autostart-Notify=true
--- gnome-screensaver-3.6.1.orig/src/gnome-screensaver.desktop.in.in
+++ gnome-screensaver-3.6.1/src/gnome-screensaver.desktop.in.in
@@ -5,7 +5,7 @@ _Comment=Launch screensaver and locker p
 Icon=preferences-desktop-screensaver
 Exec=gnome-screensaver
 OnlyShowIn=GNOME;
-AutostartCondition=GNOME3 if-session gnome-fallback
+AutostartCondition=GNOME3 unless-session gnome
 NoDisplay=true
 X-GNOME-Autostart-Phase=Application
 X-GNOME-Autostart-Notify=true
